src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,42 @@
+import "./Footer.css";
+import { footerData } from "./footerData";
+function Footer() {
+  return (
+    <footer className="footer-container p-5">
+      <ul className="list-none footer-ul p-1">
+        <li className="list-head bold">SHOP BY</li>
+        {footerData
+          .filter((links) => links.key === "shop")
+          .map((links) => (
+            <li key={links.id} className="footer-link">
+              {links.title}
+            </li>
+          ))}
+      </ul>
+
+      <ul className="list-none footer-ul p-1">
+        <li className="list-head bold">SHED</li>
+        {footerData
+          .filter((links) => links.key === "shed")
```

Comment:
we can improve the code here a little bit more, like you are comparing only `key` property so, you can use restructuring here.

Something like, 
```javascript
.filter(({key}) => key === "shed")
```

And for the `map`
```javascript
.map(({id, title}) => ...then use)
```

Reply:
Got it, but some issue is going with my github desktop in this branch. So the said changes are been made and raised in my next PR